Mathlib/Algebra/Polynomial/AlgebraMap.lean

```diff
@@ -304,6 +304,20 @@ theorem algEquivOfCompEqX_eq_iff (p q p' q' : R[X])
 theorem algEquivOfCompEqX_symm (p q : R[X]) (hpq : p.comp q = X) (hqp : q.comp p = X) :
     (algEquivOfCompEqX p q hpq hqp).symm = algEquivOfCompEqX q p hqp hpq := rfl
 
+/-- The automorphism of the polynomial algebra given by `p(X) ↦ p(a * X + b)`,
+  with inverse `p(X) ↦ p(a⁻¹ * (X - b))`. -/
+@[simps!]
+def algEquivOfCMulXAddC {R} [CommRing R] (a b : R) [Invertible a] : R[X] ≃ₐ[R] R[X] :=
```

Comment:
I don't understand the "Of" in this name

Reply:
Thanks! Renamed to `algEquivCMulXAddC`.